appcheck/firebase-appcheck/src/main/java/com/google/firebase/appcheck/internal/NetworkClient.java

```diff
@@ -130,6 +145,23 @@ public AppCheckTokenResponse exchangeAttestationForAppCheckToken(
     }
   }
 
+  /** Gets the Android package's SHA-1 fingerprint. */
+  private String getFingerprintHashForPackage() {
+    byte[] hash;
+
+    try {
+      hash = AndroidUtilsLight.getPackageCertificateHashBytes(context, context.getPackageName());
```

Comment:
nit: looks like this method is deprecated at head and `getNewestAvailablePackageCertificateRawBytes` is suggested instead, please check if it's ok to use instead.

Reply:
I tried updating the `com.google.android.gms:play-services-base` dependency to the latest version (17.6.0), but it looks like `getNewestAvailablePackageCertificateRawBytes` is not available (weirdly, only `getPackageCertificateHashBytes` is available, even though it's marked deprecated in the latest released version).